includes/models/model.llms.quiz.attempt.php

```diff
@@ -413,6 +413,7 @@ public function get_key() {
 	 *
 	 * @since 3.9.0
 	 * @since 3.16.0 Unknown.
+	 * @since [version] Added `llms_quiz_questions_randomize` and `llms_quiz_questions` filters and `include_bank_variations` parameter to questions array.
```

Comment:
```suggestion
```
We don't need a changelog here.
Why?
> Added `llms_quiz_questions_randomize` and `llms_quiz_questions` filters

this is already covered by the `@since` tag for each filter.

> and `include_bank_variations` parameter to questions array.

There's nothing here about that, probably in the add-on?

Reply:
oh! got it. Will remember next time.